docs/semgrep-app/integrations.md

```diff
@@ -1,19 +1,163 @@
 ---
 slug: integrations
 append_help_link: true
+title: Integrations
 description: "Semgrep App contains 3rd party integrations to allow users to add data from Semgrep to other tools that are part of their workflows."
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# Integrations
+# Integrating Semgrep App with third-party tools
 
-## Jira
+Semgrep App contains third-party integrations to allow you to add data from Semgrep to other tools that are part of your workflows.
 
-The Jira integration allows you to create tickets directly from the Findings page with relevant info about a particular finding.
+Currently, Semgrep App integrates with the following tools:
```

Comment:
```suggestion
Semgrep App integrates with the following tools:
```

Currently is a dangerous word in documentation, this document can get outdated very quickly... :D